meta-iot2000-example/recipes-example/iot2000setup/files/expandfs.sh

```diff
@@ -24,7 +24,7 @@ START_BLOCK=$(parted $ROOT_DEVICE -ms unit s p | grep "^2" | cut -f 2 -d: | rev
 
 parted -ms $ROOT_DEVICE rm $PART_NUMBER
```

Comment:
Hm, just wondering whether or why `parted rm` and `parted mkpart` can't be consolidated into `parted resizepart`?